PraceDomowe/Ly_HoangThien_ Homework no.5/Homework no.5 _ Final Version.ipynb

```diff
@@ -0,0 +1,1886 @@
+{
```

Comment:
<p>It is correct answer.</p><p> </p><p><code>1 point</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>